plotly/tests/test_core/test_tools/test_figure_factory.py

```diff
@@ -686,6 +686,306 @@ def test_datetime_candlestick(self):
+    #                                 {'align': 'left',
+    #                                  'font': {'color': '#000000'},
+    #                                  'showarrow': False,
+    #                                  'text': '27790000',
+    #                                  'x': 1.55,
+    #                                  'xanchor': 'left',
+    #                                  'xref': 'x1',
+    #                                  'y': 2,
+    #                                  'yref': 'y1'},
+    #                                 {'align': 'left',
+    #                                  'font': {'color': '#000000'},
+    #                                  'showarrow': False,
+    #                                  'text': 'US',
+    #                                  'x': -0.45,
+    #                                  'xanchor': 'left',
+    #                                  'xref': 'x1',
+    #                                  'y': 3,
+    #                                  'yref': 'y1'},
+    #                                 {'align': 'left',
+    #                                  'font': {'color': '#000000'},
+    #                                  'showarrow': False,
+    #                                  'text': '1980',
+    #                                  'x': 0.55,
+    #                                  'xanchor': 'left',
+    #                                  'xref': 'x1',
+    #                                  'y': 3,
+    #                                  'yref': 'y1'},
+    #                                 {'align': 'left',
+    #                                  'font': {'color': '#000000'},
+    #                                  'showarrow': False,
+    #                                  'text': '226500000',
+    #                                  'x': 1.55,
+    #                                  'xanchor': 'left',
+    #                                  'xref': 'x1',
+    #                                  'y': 3,
+    #                                  'yref': 'y1'}],
+    #                                  'height': 320,
+    #                                  'xaxis': {'dtick': 1,
+    #                                            'gridwidth': 2,
+    #                                            'showticklabels': False,
+    #                                            'tick0': -0.5,
+    #                                            'ticks': '',
+    #                                            'zeroline': False},
+    #                                  'yaxis': {'autorange': 'reversed',
+    #                                            'dtick': 1,
+    #                                            'gridwidth': 2,
+    #                                            'showticklabels': False,
+    #                                            'tick0': 0.5,
+    #                                            'ticks': '',
+    #                                            'zeroline': False}}}
+
```

Comment:
Awesome. Not sure who commented it (was it me?), but in the future, we should `skip` the whole `TestCase` with a good "skip reason" if we intend for this test to be run again, and that reason should point to an issue in the repo. Otherwise, we should delete it rather than comment it out.